pkg/actuators/machine/instances.go

```diff
@@ -333,6 +334,12 @@ func launchInstance(machine *machinev1.Machine, machineProviderConfig *awsprovid
 				}
 			}
 		}
+		metrics.RegisterFailedInstanceCreate(&metrics.MachineLabels{
```

Comment:
how are this metrics actually being exposed?
wouldn't this need to metrics.Registry.MustRegister(failedInstanceCreateCount) or anything?
https://github.com/kubernetes-sigs/controller-runtime/blob/c0438568a706ec61de31b92f4d76e7fb7e1007b9/pkg/internal/controller/metrics/metrics.go#L50

Reply:
This happens inside the init in https://github.com/openshift/machine-api-operator/pull/590/files#diff-7cbe8e056d62a2de30c7066e359bd9c9R68